YvesIraguha/personalFinanceMobile
# #171076341 Edit/Delete Expense

#### What does this PR do?
- Allow users to delete/edit expense
#### Description of Task to be completed?
-  Build an edit screen
- Enable a user to delete an expense
- Enable a user to edit an expense
#### How should this be manually tested?
- Clone the repo and checkout the branch `ft-edit-delete-expense-#171076341`
- checkout the respective branch, and run `yarn && yarn start` to start the application command
- Follow the directions provided to open the application in the simulator or real device
-  Ensure that your back-end is working. 
- Click on the info button and click on edit or delete. 
#### What are the relevant pivotal tracker stories?
#171076341
